test/api/requests/build.test.js

```diff
@@ -623,6 +623,127 @@ describe('Build API', () => {
       .catch(done);
     });
 
+    it('should update the publishedAt from processing -> success status', (done) => {
```

Comment:
Generally or these, I would like to see the `async/await` syntax used here. At least for me, it's a lot easier to see what is actually under test. With the promises, the actual method is buried in the promise chain a bit.

Happy to pair on it if you're not totally comfortable right now.

In some of these specs we are testing the effects of the request on both the site and the build. Consider either completely separating them OR making it more explicit in the spec name and code that we are testing both here.